src/document/generator.ml

```diff
@@ -26,6 +26,8 @@ let tag tag t = O.span ~attr:tag t
 let label t =
   match t with
   | Odoc_model.Lang.TypeExpr.Label s -> tag "label" (O.txt s)
+  | RawOptional _s ->
+      tag "error" (O.txt "Error: RawOptional found during rendering")
```

Comment:
That renders badly:
![2025-01-10-11-35](https://github.com/user-attachments/assets/40eb8af3-a6d0-4f4a-a95d-9354d10aaa38)

Readers of the doc can't do anything about that and we have the tag available. Why not print it the same as `Optional` ? Perhaps using the `tag "error"` but leaving the correct rendering.

Reply:
Hmm, yes, I think we can do better here.

Reply:
Now renders like this:
![2025-01-13-17-50](https://github.com/user-attachments/assets/c2300196-8dab-4179-b925-49ad5ba91ee2)